Missionframework/modules/00_init/fnc/fn_init_loadData.sqf

```diff
@@ -17,12 +17,18 @@
     BOOL
 */
 
-params [["_moduleData",[]]];
-
 if (KPLIB_param_debugSave > 0) then {diag_log "[KP LIBERATION] [SAVE] Init module loading...";};
 
+private _moduleData = [];
+
+if (!isNil "KPLIB_save_data") then {
+    _moduleData = KPLIB_save_data select {(_x select 0) == "init"};
+};
```

Comment:
As this is supposed to be also used by external modules the whole public API should be done via functions.

Something like `["init"] call KPLIB_fnc_init_getSaveData;` should be enough. The function should return module data or empty array if no module data was present in the save.

Reply:
Sounds sensible, of course 👍